src/functionalTest/resources/features/F-001 POST Hearing/S-001.3.td.json

```diff
@@ -0,0 +1,48 @@
+{
+  "_guid_": "S-001.3",
+  "_extends_" : "CreateHearingRequestBase",
+  "specs": [
+    "has the 200 OK code",
+    "an active profile in CCD",
+    "has 201 Created code",
+    "has a versionNumber of 1",
+    "has a status of HEARING_REQUESTED"
+  ],
+  "request": {
+    "body": {
+      "_extends_": "createHearingRequestFullData",
+      "partyDetails": [
+        {
+          "partyID": "P2",
+          "partyType": "ORG",
+          "partyRole": "DEF",
+          "individualDetails": null,
+          "organisationDetails": {
+            "name": "<name>",
+            "organisationType": "<organisationType>",
+            "cftOrganisationID": "<cftOrganisationID>"
+          }
+        }
+      ],
+      "caseDetails": {
+        "caseRef": "${[scenarioContext][childContexts][CreateCase][testData][actualResponse][body][id]}"
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
```

Comment:
"_extends_": "Common_201_Response",

Reply:
Done in [2188106](https://github.com/hmcts/hmc-cft-hearing-service/pull/219/commits/218810605b79e9828fd4211d2f41169c7ce26897)